test/config-next/sa.json

```diff
@@ -4,6 +4,7 @@
     "maxDBConns": 10,
     "maxConcurrentRPCServerRequests": 16,
     "debugAddr": "localhost:8003",
+    "CertStatusOptimizationsMigrated": false,
```

Comment:
Isn't that true of all `config-next` values, though? That if we port them to `config` before prod changes have been made, we'll be testing the wrong thing?